registry

```diff
@@ -524,3 +524,46 @@ erm:ERM00000506 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material I1" ;
 erm:ERM00000507 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material I2" ; dct:creator wd:Q27061849 .
 erm:ERM00000508 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material I3" ; dct:creator wd:Q27061849 .
 erm:ERM00000509 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material 14" ; dct:creator wd:Q27061849 .
+
+
+erm:ERM00000510 a obo:CHEBI_59999 ; rdfs:label "SABYDOMA use case 2 material 1" ; dct:creator wd:Q27061849 .
+erm:ERM00000511 a obo:CHEBI_59999 ; rdfs:label "SABYDOMA use case 2 material 2" ; dct:creator wd:Q27061849 .
+erm:ERM00000512 a obo:CHEBI_59999 ; rdfs:label "SABYDOMA use case 2 material 3" ; dct:creator wd:Q27061849 .
+erm:ERM00000513 a obo:CHEBI_59999 ; rdfs:label "SABYDOMA use case 2 material 4" ; dct:creator wd:Q27061849 .
+erm:ERM00000514 a obo:CHEBI_59999 ; rdfs:label "SABYDOMA use case 2 material 5" ; dct:creator wd:Q27061849 .
```

Comment:
BTW, you're now using `dct:creator wd:Q27061849` (which is fine). Can you add a line at the top with a label, and maybe link to your ORCID page?

Reply:
do you have an example ?
otherwise the link is https://orcid.org/0000-0002-4322-6179

Reply:
sure, something like:

```turtle
wd:wd:Q27061849 rdfs:label "Nina Jeliazkova"; foaf:page <https://orcid.org/0000-0002-4322-6179> .
```

(Possible with `orcid` prefix)

Reply:
See [link](https://github.com/NanoCommons/identifiers/blob/master/registry#L10-L15)

Reply:
oh I see - I can replace it with the cordis link as for other projects